app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/model/TimeRange.kt

```diff
@@ -32,7 +32,7 @@ data class TimeRange(val start: Int, val end: Int, val isOpenEnded: Boolean = fa
     fun toStringUsing(locale: Locale, range: String): String {
         val sb = StringBuilder()
         sb.append(timeOfDayToString(locale, start))
-        if (start != end || !isOpenEnded) {
+        if (start != end) {
```

Comment:
I tried to reuse as much as possible from the opening hours logic which is based on TimeRange. Currently I'm using a time range with same start and end to represent the collection time *points*. This line then ensures that "10:00" is shown in the form instead of "10:00 - 10:00". But maybe that doesn't actually work because we could have 24h range like "Mo 10:00-10:00".